ECMA2Yaml/IntellisenseFileGen/Program.cs

```diff
@@ -694,10 +649,29 @@ public static string SpecialProcessDocId(string docId)
             }
         }
 
-        static FileInfo[] GetFiles(string path, string pattern)
+        static IEnumerable<FileItem> GetFiles(string subFolder, string glob)
         {
-            DirectoryInfo di = new DirectoryInfo(path);
-            return di.GetFiles(pattern, SearchOption.AllDirectories).OrderBy(f => f.Name).ToArray();
+            var _fileAccessor = new FileAccessor(_repoRootFolder);
```

Comment:
you don't need a new FileAccessor instance since you already initialized one in main